arelle/plugin/inlineXbrlDocumentSet.py

```diff
@@ -792,6 +783,12 @@ def ixdsTargetDiscoveryCompleted(modelXbrl, modelIxdsDocument):
                 )
         modelXbrl.modelManager.loadedModelXbrls.extend(modelXbrl.supplementalModelXbrls)
 
+    # provide schema references for IXDS document
+    modelIxdsDocument.targetDocumentSchemaRefs = set()  # union all the instance schemaRefs
+    for referencedDoc in modelIxdsDocument.referencesDocument.keys():
+        if referencedDoc.type == Type.SCHEMA:
+            modelIxdsDocument.targetDocumentSchemaRefs.add(modelIxdsDocument.relativeUri(referencedDoc.uri))
+
```

Comment:
@hermfischer-wf this is still throwing an exception during save target instance in the GUI with the doc attached to #841
![Screenshot 2023-10-11 at 3 57 37 PM](https://github.com/Arelle/Arelle/assets/46454775/201c5349-291c-41a9-bc21-31f1ee4df97b)

Reply:
Unable to reproduce this.  Please slack me for more info

Reply:
The changes of this PR have been merged into PR #973 and this PR is closed.